test/object/object.cc

```diff
@@ -77,7 +82,9 @@ void DefineProperties(const CallbackInfo& info) {
 
   Boolean trueValue = Boolean::New(env, true);
   UserDataHolder* holder = new UserDataHolder();
+  UserDataHolder* functionHolder = new UserDataHolder();
```

Comment:
I think its safe to use the `holder` above rather than creating a new one.

Reply:
Yeah, I thought about that, but... didn't :smile: ... Addressed